drivers/wdf/wdfldr/class.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * PROJECT:     ReactOS WdfLdr driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     WdfLdr driver - class functions
+ * COPYRIGHT:   Copyright 2019 mrmks04 (mrmks04@yandex.ru)
```

Comment:
My comment about the names is also valid for this! (fix in all the relevant files)